### PR TITLE
VideoPress Onboarding step 0: Laying the foundation

### PR DESCRIPTION
**Pull request by pgk**
#### Proposed Changes

* Introduce the VideoPress Onboarding Flow!
* Create the first screen of the flow: the signup screen.

Screenshot 2022-07-07 at 4 24 02 PM

#### Testing Instructions

* Start your local dev and navigate to `http://calypso.localhost:3000/start/videopress`
* You are redirected to `http://calypso.localhost:3000/start/videopress/user`
* The screen looks like the pic in 1238-gh-Automattic/greenhouse

Related to #

### CODE REVIEW

File: client/signup/style.scss [hunk 1]
```diff
@@ -20,7 +20,57 @@ body.is-section-signup {
 			}
 		}
 	}
+	&.is-videopress-signup {
+		background: var( --videopress-color-background-dark );
+		color: var( --videopress-color-text );
+		letter-spacing: -0.01em;
```
Comment (roundhill) [inline client/signup/style.scss]: What's this for? 😄

File: client/signup/style.scss [hunk 2]
```diff
@@ -20,7 +20,57 @@ body.is-section-signup {
...
+				background: linear-gradient( 126.81deg, #001424 7.92%, #02223a 89.27% );
+				animation: p2-fade 5s ease-in-out infinite alternate;
+			}
+		}
 
+		.layout {
+			height: 100%;
+			display: flex;
+			flex-direction: column;
+		background: var( --videopress-color-background-dark );
```
Comment (roundhill) [inline client/signup/style.scss]: Whitespace?

File: client/assets/stylesheets/_videopress-extends.scss [hunk 3]
```diff
@@ -0,0 +1,145 @@
+/**
+ * VideoPress common styles
+ */
+
+// Text input field
+%videopress-form-input-text {
```
Comment (roundhill) [inline client/assets/stylesheets/_videopress-extends.scss:6]: I don't think I've seen the `%` character in front of the class name before. What's that do?

**Review by roundhill — CHANGES_REQUESTED** [review]
It's looking good! I noticed a few things:

The social login buttons are missing some coloring, this is from the design:
Screen Shot 2022-07-11 at 3 02 45 PM

When I hover over the `Continue with Google` button, there's some white text on white background going on: 
Screen Shot 2022-07-11 at 2 55 59 PM

File: client/assets/stylesheets/_videopress-extends.scss (hunk 3 above)
Comment (pgk) [reply client/assets/stylesheets/_videopress-extends.scss:6]: That was a TIL moment for John and I too, its https://sass-lang.com/documentation/style-rules/placeholder-selectors

File: client/signup/style.scss (hunk 1 above)
Comment (pgk) [reply client/signup/style.scss]: removed, good catch!

**Review by roundhill — APPROVED** [review]
:shipit:

**Pull request merged**